SpinnerDatePickerLib/src/main/java/com/tsongkha/spinnerdatepicker/DatePickerDialog.java

```diff
@@ -46,11 +49,15 @@
                      OnDateSetListener callBack,
                      Calendar defaultDate,
                      Calendar minDate,
-                     Calendar maxDate) {
+                     Calendar maxDate,
+                     boolean isDayEnabled,
+                     boolean updateTitleEnabled) {
```

Comment:
What about `showTitle` and `showDaySpinner` here for the parameter names? I would prefer this to enabled since this could be confused with Android View `setEnabled`

Reply:
I checked out the pull request and ran it and it seemed to work fine. Can you please also consider adding to the Espresso tests? They are in `com.tsongkha.spinnerdatepickerexample.MainActivity` and `com.tsongkha.spinnerdatepickerexample.MainActivityTest`. Ideally we would like a test for the following scenarios: `showTitle = false` leads to the title being `View.GONE` (use `ViewMatchers.effectiveVisibility`), `showDay` leads to the day spinner being `View.GONE`. It would be nice to include in these tests scrolling to make sure the scrolling still changes the date correctly in the DatePicker with these options set on.

Reply:
Also, perhaps you would like to update the README.md to show off this great new feature ;-) You could include images of the DatePicker without the title and with `showDay` set to `false`